integration/ide/bsp-server/resources/snapshots/build-targets-jvm-test-environments.json

```diff
@@ -18,14 +18,52 @@
-container-default/2.1.1/plexus-container-default-2.1.1.jar,file:/coursier-cache/https/repo1.maven.org/maven2/org/virtuslab/scala-cli/config_2.13/1.1.3/config_2.13-1.1.3.jar,file:/coursier-cache/https/repo1.maven.org/maven2/org/apache/tika/tika-core/3.1.0/tika-core-3.1.0.jar,file:/coursier-cache/https/repo1.maven.org/maven2/io/github/alexarchambault/windows-ansi/windows-ansi/0.0.6/windows-ansi-0.0.6.jar,file:/coursier-cache/https/repo1.maven.org/maven2/com/github/luben/zstd-jni/1.5.7-3/zstd-jni-1.5.7-3.jar,file:/coursier-cache/https/repo1.maven.org/maven2/io/get-coursier/cache-util/2.1.25-M13/cache-util-2.1.25-M13.jar,file:/coursier-cache/https/repo1.maven.org/maven2/javax/inject/javax.inject/1/javax.inject-1.jar,file:/coursier-cache/https/repo1.maven.org/maven2/org/codehaus/plexus/plexus-utils/4.0.1/plexus-utils-4.0.1.jar,file:/coursier-cache/https/repo1.maven.org/maven2/org/codehaus/plexus/plexus-io/3.5.0/plexus-io-3.5.0.jar,file:/coursier-cache/https/repo1.maven.org/maven2/commons-io/commons-io/2.18.0/commons-io-2.18.0.jar,file:/coursier-cache/https/repo1.maven.org/maven2/org/apache/commons/commons-compress/1.27.1/commons-compress-1.27.1.jar,file:/coursier-cache/https/repo1.maven.org/maven2/org/slf4j/slf4j-api/2.0.16/slf4j-api-2.0.16.jar,file:/coursier-cache/https/repo1.maven.org/maven2/io/airlift/aircompressor/0.27/aircompressor-0.27.jar,file:/coursier-cache/https/repo1.maven.org/maven2/org/tukaani/xz/1.9/xz-1.9.jar,file:/coursier-cache/https/repo1.maven.org/maven2/org/codehaus/plexus/plexus-classworlds/2.6.0/plexus-classworlds-2.6.0.jar,file:/coursier-cache/https/repo1.maven.org/maven2/org/apache/xbean/xbean-reflect/3.7/xbean-reflect-3.7.jar,file:/coursier-cache/https/repo1.maven.org/maven2/org/virtuslab/scala-cli/specification-level_2.13/1.1.3/specification-level_2.13-1.1.3.jar,file:/coursier-cache/https/repo1.maven.org/maven2/org/fusesource/jansi/jansi/2.4.1/jansi-2.4.1.jar,file:/coursier-cache/https/repo1.maven.org/maven2/commons-codec/commons-codec/1.17.1/commons-codec-1.17.1.jar,file:/coursier-cache/https/repo1.maven.org/maven2/org/apache/commons/commons-lang3/3.16.0/commons-lang3-3.16.0.jar,file:/mill-workspace/out/core/coursierutil/compile.dest/classes/,file:/mill-workspace/out/core/define/compile.dest/classes/,file:/mill-workspace/out/core/constants/buildInfoResources.dest/,file:/mill-workspace/out/core/constants/compile.dest/classes/,file:/mill-workspace/out/core/api/buildInfoResources.dest/,file:/mill-workspace/out/core/api/compile.dest/classes/,file:/mill-workspace/out/core/util/buildInfoResources.dest/,file:/mill-workspace/out/core/util/compile.dest/classes/,file:/mill-workspace/out/libs/scalalib/api/buildInfoResources.dest/,file:/mill-workspace/out/libs/scalalib/api/compile.dest/classes/,file:/mill-workspace/out/libs/testrunner/entrypoint/compile.dest/classes/,file:/mill-workspace/out/libs/testrunner/compile.dest/classes/,file:/mill-workspace/out/libs/scalalib/compile.dest/classes/,file:/mill-workspace/libs/scalalib/resources/",
```

Comment:
Regarding the single slash after `file:`: Could be a missing `sanitizeUrl` call, that should ensure the same formatting as everywhere else.